descriptive-link-5effbb.md

```diff
@@ -0,0 +1,179 @@
+---
+id: 5effbb
+name: Link text is descriptive
+rule_type: atomic
+
+description: | 
+ This rule checks that all links describe their purpose
+
+accessibility_requirements: 
+  wcag20:2.4.4: # Link Purpose (In Context) 
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+
+input_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Carlos Duarte
+- Marie Trudelle 
+- Ramit Garg
+---
+
+## Applicability
+
+The rule applies to any HTML element with the [semantic role](#semantic-role) of `link` that has an [accessible-name](#accessible-name) that is not the empty string ("").
+
+## Expectation
+
+Both the [text content](#text-content), if [visible](#visible), and the [accessible-name](#accessible-name) of the target element, together with its [visible](#visible) [context](#context) describe the purpose of the link.
+
+Note: Even though elements with the [semantic role](#semantic-role) of `link` [support name from content](https://www.w3.org/TR/wai-aria-1.1/#namefromcontent), their [text content](#text-content) and [accessible-name](#accessible-name) may be different if the content's author decides to. For this reason, this rules checks both [text content](#text-content) and [accessible-name](#accessible-name).
+
+## Assumptions
+
+- The rule assumes that all links are [user interface components](https://www.w3.org/TR/WCAG20/#user-interface-componentdef) as defined by WCAG 2.
```

Comment:
Why is this relevant? Isn't this only relevant for SC 4.1.2?

Reply:
True! I'm removing it.